sdk/src/ix.ts

```diff
@@ -418,18 +418,30 @@ export class WhirlpoolIx {
   }
 
   /**
+   *
    * A set of transactions to collect all fees and rewards from a list of positions.
    *
    * @param ctx - WhirlpoolContext object for the current environment.
    * @param params - CollectAllPositionAddressParams object.
    * @param refresh - if true, will always fetch for the latest values on chain to compute.
    * @returns
    */
-  public static collectAllForPositionsTxns(
+  public static async collectAllForPositionsTxns(
     ctx: WhirlpoolContext,
     params: ix.CollectAllPositionAddressParams,
     refresh: boolean
   ) {
     return ix.collectAllForPositionAddressesTxns(ctx, params, refresh);
   }
+
+  /**
+   * Collect protocol fees from a list of pools
+   *
+   * @param ctx - WhirlpoolContext object for the current environment.
+   * @param poolAddresses the addresses of the Whirlpool accounts to collect protocol fees from
+   * @returns A transaction builder to resolve ATA for tokenA and tokenB if needed, and collect protocol fees for all pools
+   */
+  public static async collectProtocolFeesForPools(ctx: WhirlpoolContext, poolKeys: Address[]) {
+    return ix.collectProtocolFees(ctx, poolKeys);
+  }
```

Comment:
If we were to put collectProtocolFees in either WhirlpoolClient or WhirlpoolIx, I think WhirlpoolClient is good place. This operation appears to be an advanced operation equivalent to WhirlpoolClient.collectFeesAndRewardsForPositions.